### PR TITLE
feat(zboxapi): added zboxapi client sdks

### PR DESCRIPTION
**Pull request by cnlangzi**
### Changes
- added zboxapi client package
- exposed zboxapi client on windows/android/ios/wasm sdks.

### Fixes
-

### Tests
Tasks to complete before merging PR:
- [ ]  Ensure system tests are passing. If not [Run them manually](https://github.com/0chain/gosdk/actions/workflows/system_tests.yml) to check for any regressions :clipboard:
- [ ]  Do any new system tests need added to test this change? do any existing system tests need updated? If so create a PR at [0chain/system_test](https://github.com/0chain/system_test)
- [ ]  Merge your system tests PR to master AFTER merging this PR

### Associated PRs (Link as appropriate):
- 0box: https://github.com/0chain/0box/pull/276

### CODE REVIEW

File: core/conf/config.go [hunk 1]
```diff
@@ -63,6 +63,11 @@ type Config struct {
 
 	// Ethereum node: "https://ropsten.infura.io/v3/xxxxxxxxxxxxxxx"
 	EthereumNode string `json:"ethereum_node,omitempty"`
+
+	// ZboxApi 0box api host host: "https://0box.dev.0chain.net"
+	ZboxApi string `json:"zbox_api"`
```
Comment (peterlimg) [inline core/conf/config.go]: If this is service host address, then let's name it `ZboxHost` or `ZboxAddr`

**Review by peterlimg — CHANGES_REQUESTED** [review]
We can use zbox as package name, the suffix `api` is not necessary as all the stuff in the gosdk are SDKs.

File: core/conf/config.go (hunk 1 above)
Comment (cnlangzi) [reply core/conf/config.go]: > We can use zbox as package name, the suffix `api` is not necessary as all the stuff in the gosdk are SDKs.

we already have a `zbox` package. it is a wrapper of zbox cli. `zboxapi` is different from it. it is client sdk of github.com/0chain/0box.  
do you think they should be merged into one package?

Comment (cnlangzi) [reply core/conf/config.go]: @peterlimg  updated it with `ZboxHost`

Comment (peterlimg) [reply core/conf/config.go]: okay, then let's keep it as `zboxapi`

**Review by peterlimg — APPROVED** [review]
LGTM

**Pull request merged**